datafusion/core/src/physical_plan/file_format/parquet.rs

```diff
@@ -414,8 +414,7 @@ impl FileOpener for ParquetOpener {
                     table_schema.as_ref(),
                     builder.metadata(),
                     reorder_predicates,
-                    &file_metrics.pushdown_rows_filtered,
-                    &file_metrics.pushdown_eval_time,
+                    &file_metrics,
```

Comment:
it was just bothering me for some reason that this was different than `prune_row_groups` and `build_page_filter`

Reply:
Keep align is necessary. Will keep in mind in future.

Reply:
(to be clear I was the one who wrote the original code that was not consistent -- so I need to keep it in mind :) )